gravitee-apim-repository/gravitee-apim-repository-gateway-bridge-http/gravitee-apim-repository-gateway-bridge-http-server/src/main/java/io/gravitee/repository/bridge/server/http/configuration/HttpServerConfiguration.java

```diff
@@ -75,6 +75,9 @@ public class HttpServerConfiguration {
     @Value("${services.bridge.http.maxChunkSize:8192}")
     private int maxChunkSize;
 
+    @Value("${services.bridge.http.workerPoolSize:-1}")
```

Comment:
I would have put io.vertx.core.VertxOptions.DEFAULT_WORKER_POOL_SIZE as default value (i.e. 20).
From a user perspective, I would understand the -1 as either no limit or no worker

Reply:
As per our discussion let's go with -1 for now